tests/testUtils/MockNetwork.ts

```diff
@@ -0,0 +1,177 @@
+/* eslint-disable no-restricted-syntax */
+import nock from 'nock';
```

Comment:
`nock` provides a way to mock the network during tests. It intercepts network calls and returns responses as configured below. I've configured it to return responses modeled after Esplora.